robot/basestation/static/js/roslibjs/ros-helpers.js

```diff
@@ -2,6 +2,17 @@ REQUEST_TIMEOUT = 3000
 ROTATE_TIMEOUT = 1000
 lastRotate = 0
 
+// minimum and maximum acceptable battery voltages (volts)
+MIN_VOLTAGE = 12.5
+MAX_VOLTAGE = 16.8
+VOLTAGE_LEEWAY = 0.5
+// minimum and maximum acceptable battery tempratures (degrees celcius)
+MIN_TEMP = 0
+MAX_TEMP = 85
+TEMP_LEEWAY = 1
+// variable to toggle unacceptable voltage and temperature indicators
+ALERT_ENABLE = false;
```

Comment:
shouldn't this be true by default? also CS --> no semicolons

Reply:
im small brain